stl/inc/__msvc_iter_core.hpp

```diff
@@ -34,43 +34,29 @@ _EXPORT_STD struct contiguous_iterator_tag : random_access_iterator_tag {};
 
 template <class _Ty>
 concept _Dereferenceable = requires(_Ty& __t) {
-    { *__t } -> _Can_reference;
-};
+                               { *__t } -> _Can_reference;
+                           };

```

Comment:
Is this correct? It seems not quite right.

Reply:
It looks strange at first, but it appears to be intentional behavior from clang-format. What it's doing is considering the `requires` to be where the indentation should begin, so that the body `{ *__t } -> _Can_reference;` is indented one level beyond that, and the closing `};` is indented at the same level as the `requires`.

Reply:
Interesting... Seems fair enough.